lib/virt_autotest/utils.pm

```diff
@@ -150,11 +157,13 @@ sub create_guest {
     my $location     = $guest->{location};
     my $autoyast     = $guest->{autoyast};
     my $macaddress   = $guest->{macaddress};
-    my $extra_params = $guest->{extra_params} // "";
+    my $on_reboot    = $guest->{on_reboot}    // "destroy";    # configurable on_reboot policy
+    my $extra_params = $guest->{extra_params} // "";           # extra-parameters
+    my $extra_args   = get_var("VIRTINSTALL_EXTRA_ARGS", "") . " " . get_var("VIRTINSTALL_EXTRA_ARGS_" . uc($name), "");
```

Comment:
Why is there ` . uc($name), "")`? Maybe we can have a Jitsi call about it.

Reply:
Using `clone_job`, all settings get UPPERCASED, but our guest names are lowercase. So, without this `uc` you would not be able to get the setting that you passed via clone-job.

To me it looks like, that settings are just expected to be uppercase, so we just do that here.

Reply:
AH I got it, this is a guest specifi setting! OK, sorry :)